package.json

```diff
@@ -50,6 +50,7 @@
     "jquery-validation": "^1.19.0",
     "katex": "^0.13.2",
     "lodash": "^4.17.19",
+    "micromodal": "^0.4.6",
```

Comment:
There should be a minor provision bump as well since we are adding a dependency.